include/seastar/core/reactor.hh

```diff
@@ -502,6 +502,8 @@ public:
     future<> touch_directory(std::string_view name, file_permissions permissions = file_permissions::default_dir_permissions) noexcept;
     future<std::optional<directory_entry_type>>  file_type(std::string_view name, follow_symlink = follow_symlink::yes) noexcept;
     future<stat_data> file_stat(std::string_view pathname, follow_symlink) noexcept;
+    future<> change_ownership_of_file(std::string_view filepath, uint64_t groupid) noexcept;
+    future<group_details> grp_detail(std::string_view groupname) noexcept;
```

Comment:
I guess this is getgrnam() so I would call it that name.
But I have to wonder - this isn't even a system call - this is just one of a bazillion libc functions, do we really need a version of each and every one of them in Seastar? 
Maybe at some point we should stop adding all these random functions and add a conventient API for running (via _thread_pool) *any* potentially-sleeping library function?

Reply:
We have seastar::alien already.

Reply:
Looks like there are other libc functions, which will need to be cleaned up as well, can work on this as a followup.